shell/browser/api/atom_api_system_preferences_mac.mm

```diff
@@ -503,19 +503,23 @@ AVMediaType ParseMediaType(const std::string& media_type) {
   return AXIsProcessTrustedWithOptions((CFDictionaryRef)options);
 }
 
-std::string SystemPreferences::GetColor(const std::string& color,
-                                        mate::Arguments* args) {
+std::string SystemPreferences::GetColor(gin_helper::ErrorThrower thrower,
+                                        const std::string& color) {
   NSColor* sysColor = nil;
   if (color == "alternate-selected-control-text") {
     sysColor = [NSColor alternateSelectedControlTextColor];
+    EmitDeprecationWarning(env, "Use selected-content-background instead.",
+                           "electron");
   } else if (color == "control-background") {
     sysColor = [NSColor controlBackgroundColor];
   } else if (color == "control") {
     sysColor = [NSColor controlColor];
   } else if (color == "control-text") {
     sysColor = [NSColor controlTextColor];
-  } else if (color == "disabled-control") {
+  } else if (color == "disabled-control-text") {
```

Comment:
How long has this been wrong? Could someone have figured out the wrong value and should we specifically support the old value with a deprecation warning similar to above to be nicer to those folks?

Reply:
(resolved offline)